nextcloud/android
# Show maintenance info

Fix #989

We have on nearly all occasions a snackbar, when server is in maintenance mode (e.g. renaming, delete files).
But if you browse into a subfolder or go back, you do not get an info, but outdated/none folder content.

Please note that there is no automatic/in background check if server is available again.

Steps to test:
- open up app w/o server in maintenance mode
- browse
- set server in maintenance mode
- browse, see info on top
- try other operations (CRUD on files, share, remove share)

![2018-07-05-101525](https://user-images.githubusercontent.com/5836855/42310690-5cc55ec4-803c-11e8-8532-32d22d03937a.png)

The first part (extension of Toolbar) was done by @AndyScherzinger :tada:.
